tests/mol-audio_no_clipbegin/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System starts at the beginning of the audio track in the absence of a clipBegin attribute in the smil file</dc:description>
+    <dc:identifier id="pub-id">mol-audio_no_clipbegin</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Automatic start at the beginning of an audio track</dc:title>
```

Comment:
Optional, but consider making the title reference clipBegin to match the clipEnd test above.

Reply:
Chaned